packages/iov-keycontrol/src/userprofile.ts

```diff
@@ -50,22 +85,64 @@ export interface UserProfileOptions {
  * UserProfile via the UserProfileController to get an unlocked UserProfile.
  */
 export class UserProfile {
+  /**
+   * Derives an encryption key from the password. This is a computationally intense task that
+   * can take many seconds.
+   *
+   * Use this function to cache the encryption key in memory.
+   *
+   * @param formatVersion Set this if you got a UserProfileEncryptionKeyUnexpectedFormatVersion error. This
+   * error usually means a profile was encrypted with an older format version.
+   */
+  public static async deriveEncryptionKey(
+    password: string,
+    formatVersion?: number,
+  ): Promise<UserProfileEncryptionKey> {
+    const version = formatVersion !== undefined ? formatVersion : latestFormatVersion;
```

Comment:
How about using a default value instead?

Reply:
I am very much surprised that I can use the module runtime constant `latestFormatVersion` as part of the function signature. But for some reason it works, great.